demos/python_demos/llm_text_generation/README.md

```diff
@@ -1,14 +1,27 @@
-# Llama demo with python node {#ovms_demo_python_llama}
+# LLM text generation with python node {#ovms_demo_python_llm_text_generation}
+
+This demo shows how to take advantage of OpenVINO Model Server to generate content remotely with LLM models. 
+The server is running here the user provided python code in a MediaPipe Python calculator. We manage the generation with Hugging Faces pipeline and OpenVINO Runtime as the execution engine.
+The generation cycles are configured in the MediaPipe graph. Two use cases are possible:
+- with unary calls - when the client is sending a single prompt to the graph and receives a complete generated response
+- with gRPC streaming - when the client is sending a single prompt the graph and receives a stream of responses
+
+The unary calls are simpler but the response might be sometimes slow when many cycles are needed on the server side
+
+The gRPC stream is a great feature when more interactive approach is needed allowing the user to read the response as they are getting generated.
+
+This demo is presents the use case with `red-pajama-3b-chat` model but the included python scripts are prepared for several other LLM  models like `Llama-2-7b-chat-hf` and TBD.
```

Comment:
is presents -> presents

Do we want to add "TBD" and "models like"? Maybe let's just stick with what we have working.

Reply:
done